### ISSUE TITLE
Variables beginning with the letters "as" cause an error in Cicero parse.

### ISSUE THREAD

**Issue by adrianffletcher** [accordproject/cicero#407]
**Describe the bug**
If a Cicero template is created with a variable name that begins with the letter "as" then Cicero parse will give the following error:
`Unexpected varas token: "as"`

**To Reproduce**
1. Start with the helloworld template.
2. Change the variable "name" to "assetName" across all files.
3. Run cicero parse.
4. See the following error: `Unexpected varas token: "as"`

**Expected behavior**
Cicero parse should succeed.

**Comment by adrianffletcher** [comment]
Attached example:
[helloasset.zip](https://github.com/accordproject/cicero/files/3555059/helloasset.zip)

**Comment by jeromesimeon** [comment]
Thanks for the report.

This issue also affects the dynamic template branch:
```
bash-3.2$ more grammar/template.tem 
Name of the person to greet: {{assetName}}.
Thank you!
grammar/template.tem (END)
bash-3.2$ ~/git/cicero/packages/cicero-cli/index.js parse 
10:30:40 - info: Using current directory as template folder
10:30:40 - info: Loading a default sample.txt file.
10:30:40 - error: invalid syntax at line 1 col 32:

  Name of the person to greet: {{as
                                 ^
Unexpected varas token: "as"
```

**Comment by jeromesimeon** [comment]
I think this is an example of: https://github.com/no-context/moo#keywords
and should be fixed using the `keywords` function, as explained at the end of that link

**Comment by jeromesimeon** [comment]
Fixed in #409 

A sample run with the attached archive gives:
- With Cicero 0.13.4:
```
bash-3.2$ cicero --version
0.13.4
bash-3.2$ cicero parse 
15:26:52 - info: Using current directory as template folder
15:26:52 - info: Loading a default sample.txt file.
15:26:53 - error: invalid syntax at line 1 col 32:

  Name of the person to greet: [{as
                                 ^
Unexpected varas token: "as"
```
- With the PR:
```
bash-3.2$ ~/git/cicero/packages/cicero-cli/index.js parse 
15:27:08 - info: Using current directory as template folder
15:27:08 - info: Loading a default sample.txt file.
15:27:09 - info:
{
  "$class": "org.accordproject.helloworld.HelloWorldClause",
  "clauseId": "1776225e-aa6d-41a5-809a-c14dcdfa1807",
  "assetName": "Fred Blogs"
}
```

**Comment by jeromesimeon** [comment]
Fixed in PR #409. Should go out with the next release.
